roxctl/common/npguardutils/netpolutils.go

```diff
@@ -0,0 +1,36 @@
+package npguardutils
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+
+	e "github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/errox"
+)
+
+var (
+	// ErrNPGErrorsIndicator errors indicator message
+	ErrNPGErrorsIndicator = errors.New("there were errors during execution")
```

Comment:
When thinking about package and functions name, we need to imagine how this will be used - calling it `npguardutils.ErrNPGErrorsIndicator` has the following problems:
- we mention npguard twice
- we mention utils but this is actually an error

Reply:
When thinking how this could be used, the following comes to my mind:

1. `npg.ErrErrorsIndicator`
2. `npguard.ErrFoundErrors`
3. or maybe `npg.ErrErrors`?

Reply:
done, moved to the `common/npg` package